src/pipes/validate.pipe.ts

```diff
@@ -11,7 +11,16 @@ export class ValidationPipe implements PipeTransform<any> {
         if (!metatype || !this.toValidate(metatype)) {
             return value
         }
+
+        // Convert numeric strings back to numbers
+        for (const key in value) {
+            if (typeof value[key] === 'string' && !isNaN(Number(value[key]))) {
+                value[key] = Number(value[key])
+            }
+        }
```

Comment:
look good but u can try https://github.com/nestjs/nest/issues/1331

Reply:
https://github.com/nestjs/nest/issues/1331 is it way not working?

Reply:
Yes, I have try it and still not working.

Reply:
Ok, I got it.
I will search another way to solve this problem